### ISSUE TITLE
How to avoid overfitting when doing text classification?

### ISSUE THREAD

**Issue by Liwink** [facebookresearch/fastText#275]
It is amazing to use fastText to classify documents for its speed and accuracy.
But I have no idea about how to avoid overfitting effectively.
When I use bigrams and train more epochs, the training accuracy is greater, but the validation accuracy is lower.

How can I avoid overfitting effectively when using fastText to learn a text classifier?
Thanks.

**Comment by xharrietx411** [comment]
[WhatsApp Chat - Luke Comley.zip](https://github.com/facebookresearch/fastText/files/1150420/WhatsApp.Chat.-.Luke.Comley.zip)
![img_3739](https://user-images.githubusercontent.com/23618273/28240368-a247daa2-6978-11e7-9ad5-1956cd78b515.PNG)

**Comment by xharrietx411** [comment]
???

**Comment by cpuhrsch** [comment]
Hello @Liwink ,

Thank you for your post. You might find more support for this kind of issue within one of our community boards. In particular, the Facebook group has a lot of members many of whom are ML experts and are keen on discussing applications of this library.

Specifically, there is a

[Facebook group](https://www.facebook.com/groups/1174547215919768/)
[Stack Overflow tag](https://stackoverflow.com/questions/tagged/fasttext)
[Google group](https://groups.google.com/forum/#!forum/fasttext-library)

If you do decide to move this to one of our other community boards, please consider closing this issue.

Thanks,
Christian

**Comment by AbhiPawar5** [comment]
Are there any new ideas to avoid overfitting with fasttext now? @Liwink @xharrietx411
